build-system/tasks/e2e/index.js

```diff
@@ -50,6 +51,13 @@ async function setUpTesting_() {
     await buildRuntime();
   }
 
+  await buildBentoE2E({
```

Comment:
Move this inside the `if(!argv.nobuild)` conditional

Reply:
I talked a bit with @danielrozenberg. I think we want this to get build with every e2e test, as opposed to with every dist/build because the e2e tests are ran less frequently then the dist/build commands.

Reply:
That's fine, but there should still be the option to skip the build when running locally.

Reply:
If you're running it locally, you'll want it to rebuild or else your changes won't be picked up.

The e2e fixture is a bundle of the entry point and the component being tested.